apiserver/facades/schema.json

```diff
@@ -14278,16 +14278,16 @@
                         "error": {
                             "$ref": "#/definitions/Error"
                         },
-                        "host-key": {
+                        "jump-server-host-key": {
```

Comment:
As I mentioned above, we'll want to send the known host line, and the fingerprint (probably MD5 and SHA256). Here is how JIMM returns the fingerprints btw https://jimm.k8s.dev.canonical.com/ssh/public-key-fingerprints

Reply:
Gotcha